### PR TITLE
fixed issue #136 , create folder model now automatically gain focus

### PR DESCRIPTION
**Pull request by kasvith**
# Pull Request for Issue #136  .

### Summary of Changes
- Updated package.json to use vue-focus module
- Create folder model gains focus when it shown

### Expected result
When create new folder button clicked, text input gains the access by default

### CODE REVIEW

**Comment by kasvith** [comment]
#139 closing due to merge conflict

**Comment by laoneo** [comment]
No need to close it. You can fix the merge conflicts.

**Comment by kasvith** [comment]
@laoneo I can't fix it because I don't have write access, just need to add following to package.json

`"author": "",`

I've missed that

**Comment by kasvith** [comment]
@laoneo can you give me write access ?

**Comment by kasvith** [comment]
from second commit I've fixed the merge conflict

**Review by dneukirchen — COMMENTED** [review]
Good Idea @kasvith . We should set the focus, but im not sure if we really need an additional package for this. Please dont get me wrong: i dont want to reinvent the wheel, but we should add external deps only when we really need them.

I think this can be resolved without an external package with a reference to the input element:

```
<input type="text" id="folder" ref="folder" ...>
```

```
created() {
    this.$refs.folder.focus();
}
```

**Comment by laoneo** [comment]
Its your fork, you should be able to do that by your own. No need to get permissions on this repo.

**Comment by kasvith** [comment]
@dneukirchen we can use your method if the create-folder-model being created every time when user click the **Create Folder** button, actually it is created once and hidden until it is being called, in that case the created method will fire only 1 time, so we need to trigger this focus event after the `v-if` passed, for that this simple library helps

**Comment by dneukirchen** [comment]
True. 
Can you resolve the conflicts, so that i can merge this one?

**Comment by kasvith** [comment]
how to resolve this ?
this one keeps showing up

image

**Comment by kasvith** [comment]
in my fork it is been resolved, also in second commit i've fixed that, but it keep showing up

**Comment by kasvith** [comment]
@dneukirchen resolved merge conflicts

File: administrator/components/com_media/package.json [hunk 1]
```diff
@@ -1,4 +1,34 @@
...
+		"node-sass": "^4.5.1",
+		"vueify": "^9.4.1"
+	},
+	"browserify": {
+		"transform": [
+			"vueify",
+			"babelify"
+		]
+	}
+
```
Comment (dneukirchen) [inline administrator/components/com_media/package.json]: Please remove duplicate code and indentation (2 whitespaces)

Comment (dneukirchen) [reply administrator/components/com_media/package.json]: and kindly run "npm run build" in the media manager root to add the updated mediamanager.js file.

**Review by dneukirchen — CHANGES_REQUESTED** [review]

File: administrator/components/com_media/package.json (hunk 1 above)
Comment (kasvith) [reply administrator/components/com_media/package.json]: sure

Comment (kasvith) [inline administrator/components/com_media/package.json]: @dneukirchen do we want to add mediamanager.js file ? it was not changed

**Comment by dneukirchen** [comment]
If you need any help, just ask me on glip. The package.json is not formatted correctly and has a syntax error (closing } missing). Thats the reason why npm run build throws an error instead of building the updated assets.

**Comment by kasvith** [comment]
@dneukirchen can you add me to Joomla glip ?

**Comment by dneukirchen** [comment]
pls create an accolunt on https://glip.com/ i will add you

**Comment by kasvith** [comment]
@dneukirchen can you check now?

**Comment by kasvith** [comment]
updated and fixed all bugs

**Review by dneukirchen — APPROVED** [review]

**Comment by dneukirchen** [comment]
thank you @kasvith

**Comment by laoneo** [comment]
Good work!

**Pull request merged**